api/v1/server/oas/transformers/v1/tasks.go

```diff
@@ -257,6 +259,8 @@ func ToTask(taskWithData *sqlcv1.PopulateSingleTaskRunDataRow, workflowRunExtern
 		parsedWorkflowRunUUID = &id
 	}
 
+	stepId := uuid.MustParse(sqlchelpers.UUIDToStr(taskWithData.StepID))
```

Comment:
why are we casting and parsing here..? are we trying to validate?

Reply:
we're doing this in a few places it seems

Reply:
I think some of this is new to the OLAP repo because of moving off Prisma (@abelanger5 let me know if this is wrong).

we need to do this to convert from a `pgtype.UUID` to a `uuid.UUID` - it's a bit clunky